src/Components/Admin/Assets/fileTypes.ts

```diff
@@ -1,5 +1,5 @@
 export enum FileTypes {
-    GLTF = "gltf",
-    GLB = "glb",
+    GLTF = ".gltf",
+    GLB = ".glb",
```

Comment:
I made this change so I can use these directly as file type restrictions on line 254 in uploadAssetModal.js

Reply:
BE already adds a dot to the extension, so FE just needs to send gltf or glb, rather than .gltf or .glb. That's the cause of the error that @amoliksingh posted in his loom video